buildSrc/src/main/groovy/local.java.container-spring-conventions.gradle

```diff
@@ -0,0 +1,31 @@
+/**
+ * Provides docker container settings
+ * Based on https://github.com/department-of-veterans-affairs/lighthouse-di-starter-boot/blob/main/buildSrc/src/main/groovy/starter.java.container-spring-conventions.gradle
+ */
+
+plugins {
+    id 'org.springframework.boot'
+    id 'local.java.container-conventions'
+}
+
+docker {
+    files "build/libs/${bootJar.archiveFileName.get()}", "bin/entrypoint.sh"
+    buildArgs([JAR_FILE: bootJar.archiveFileName.get(), ENTRYPOINT_FILE: "entrypoint.sh"])
+}
+
+dockerRun {
+    env 'JAVA_PROFILE': '-Dspring.profiles.include=docker'
+}
+
+tasks.dockerPrepare.configure {
+    dependsOn(tasks.bootJar)
+}
+
+bootJar {
+    manifest {
+        attributes(
+                'Application-Version': "${project.version}",
+                'Build-Timestamp': new java.text.SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ").format(new Date())
+        )
+    }
+}
```

Comment:
Do we still need all the other .example files in buildSrc ?

Reply:
Yes, for now, let's keep them for reference. Folks aren't familiar with how to use them yet. Individual `example` files can be replaced with `local` files as we update them against the latest versions at https://github.com/department-of-veterans-affairs/lighthouse-di-starter-boot/tree/main/buildSrc/src/main/groovy AND customize them. If we create a wiki page to describe their usage, then the example files can be removed.